[2024-08-28T04:11:06.747+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-28T04:11:06.961+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: movies_dag.subir_data manual__2024-08-28T04:10:54.475795+00:00 [queued]>
[2024-08-28T04:11:06.972+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: movies_dag.subir_data manual__2024-08-28T04:10:54.475795+00:00 [queued]>
[2024-08-28T04:11:06.974+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 4
[2024-08-28T04:11:06.991+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): subir_data> on 2024-08-28 04:10:54.475795+00:00
[2024-08-28T04:11:07.000+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=136) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-28T04:11:07.003+0000] {standard_task_runner.py:72} INFO - Started process 138 to run task
[2024-08-28T04:11:07.003+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'movies_dag', 'subir_data', 'manual__2024-08-28T04:10:54.475795+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl.py', '--cfg-path', '/tmp/tmpd39t78uo']
[2024-08-28T04:11:07.006+0000] {standard_task_runner.py:105} INFO - Job 40: Subtask subir_data
[2024-08-28T04:11:07.064+0000] {task_command.py:467} INFO - Running <TaskInstance: movies_dag.subir_data manual__2024-08-28T04:10:54.475795+00:00 [running]> on host e1752bf327c8
[2024-08-28T04:11:07.155+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='pedro' AIRFLOW_CTX_DAG_ID='movies_dag' AIRFLOW_CTX_TASK_ID='subir_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-28T04:10:54.475795+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-28T04:10:54.475795+00:00'
[2024-08-28T04:11:07.156+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-28T04:11:07.168+0000] {dag_intermediario.py:48} ERROR - ('Error when uploading data: ', TypeError("'NoneType' object is not subscriptable"))
[2024-08-28T04:11:07.169+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-08-28T04:11:07.179+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-28T04:11:07.180+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=movies_dag, task_id=subir_data, run_id=manual__2024-08-28T04:10:54.475795+00:00, execution_date=20240828T041054, start_date=20240828T041106, end_date=20240828T041107
[2024-08-28T04:11:07.219+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-28T04:11:07.235+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-28T04:11:07.238+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
